app/backend/src/couchers/servicers/api.py

```diff
@@ -899,9 +898,11 @@ def get_num_references(session, user_ids):
     )
 
 
-def user_model_to_pb(db_user, session, context):
+def user_model_to_pb(db_user, session, context, is_admin_should_show_invisible=False):
     # note that this function should work also for banned/deleted users as it's called from Admin.GetUser
     # note that this function is sometimes called by a logged out user, in which case context comes from make_logged_out_context
+    caller_is_admin = is_admin_should_show_invisible
```

Comment:
You used to have another check here. Could you please add it back as a temporary check: I'm a little antsy about this change, so if you add back the check that makes sure those users are only ever shown if the user is admin and API is admin call, that would assuade the fears.